homeassistant/components/demo/strings.json

```
@@ -1,5 +1,21 @@
 {
   "title": "Demo",
+  "issues": {
+    "out_of_blinker_fluid": {
+      "title": "The blinker fluid is empty and needs to be refilled",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "Blinker fluid needs to be refilled",
+            "description": "Press OK when blinker fluid has been refilled"
+          }
+        }
+      }
+    },
+    "transmogrifier_deprecated": {
+      "title": "The transmogrifier component is deprecated"
```

```suggestion
      "title": "The transmogrifier component is deprecated",
      "description": "The transmogrifier component is now deprecated due to the lack of local control available in the new API"
```

Can we make descriptions mandatory for non-fixables issues?

---

Do you mean we should enforce it in hassfest?

---

Yea I think it would be necessary to describe what to do to fix the issue

---

Extended the hassfest check in [daf04a4](https://github.com/home-assistant/core/pull/75081/commits/daf04a4c16a60298f972285e533361a00d8c7088)